evap/evaluation/templates/infobox.html

```diff
@@ -9,6 +9,11 @@
                     aria-label="Close"></button>
         </div>
         <div class="callout-content small">
+            {# set infobox to hidden, if user closed it before #}
+            <script>
+                if (localStorage["infobox_{{ infotext.page }}"] === "hide")
+                    document.querySelector("#{{ infotext.page }}").classList.add("closed");
+            </script>
```

Comment:
How much earlier than the code in `infobox.ts` does this run?

Reply:
The important part here is, that the browser executes the javascript before the browser parses/displays the content of the content-div, so the infobox is closed via css before the browser displays the content of the infotext. This is only possible with scripts that aren't modules.
(Tested with firefox)